templates/web/base/admin/bodies/contact-form.html

```diff
@@ -68,7 +68,7 @@
         <label for="updates_disallowed">[% loc('Disable updates on reports in this category') %]</label>
     </p>
 
-    [% IF body.send_method == 'Open311' %]
+    [% IF body.send_method == 'Open311' OR (body.can_be_devolved AND contact.send_method == 'Open311') %]
```

Comment:
Could `contact.sent_by_open311` be used here now? (Also, nice catch of something I'd not considered when adding the protection field!)